### DISCUSSION TITLE
How can I make my buttons whiter as in daisyUI v2?

### DISCUSSION THREAD

**Discussion by jaimeiniesta** [saadeghi/daisyui#2309] [Q&A] [answered]
Hi there!

I'm upgrading daisyUI to 3.6.4, everything went smoothly except that now the color in the buttons is less white, more gray. This is a primary button in version 2:

<img width="216" alt="Captura de pantalla 2023-08-30 a las 19 25 36" src="https://github.com/saadeghi/daisyui/assets/2629/241dee1d-2d4a-4d9d-9f4c-bba1701e0dbf">

and this is is how it looks after upgrading to daisyUI 3.6.4:

<img width="211" alt="Captura de pantalla 2023-08-30 a las 19 25 42" src="https://github.com/saadeghi/daisyui/assets/2629/f410e1a8-0a73-46f6-a23b-475728de0812">

I think it looked better before. What's the best way to keep the button colors as they were before?

Thanks!

**Comment by saadeghi** [comment, marked as answer]
The text color is the `*-content` color. You can customize it by [customizing an existing theme](https://daisyui.com/docs/themes/#-7) or in your [custom theme](https://daisyui.com/docs/themes/#-4) by adding the content color for it.  
For example if that button is `primary` button, you can set `"primary-content": "#FFFFFF"` in your theme to make the text color white.  

If you don't set `primary-content`, it would be generated automatically. In daisyUI 2 the algorithm used to generate only white/black. In daisyUI 3, the algorithm creates a mix from the background color + white/black with a good contrast to make it look more visually appealing.

Let me know if you have any questions.
